spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'menu'
+require 'order'
+
+describe 'user stories' do
+
+# As a customer
+# So that I can check if I want to order something
+# I would like to see a list of dishes with prices
+  describe 'customer can see a list of dishes & prices' do
+    it 'displays the menu' do
+      menu = Menu.new
+      expect(menu.display_menu).to be_a(Hash)
+    end
+  end
+
+# As a customer
+# So that I can order the meal I want
+# I would like to be able to select some number of several available dishes
+  describe 'customer can choose items from a menu' do
+    it 'allows customer to make an order list' do
+      order = Order.new(Menu.new)
+      order.read_menu
+      order.choose("Margherita")
+      expect(order.list).to include("Margherita" => 6)
+    end
+
+  end
+
+# As a customer
+# So that I can verify that my order is correct
+# I would like to check that the total I have been given matches the sum of the various dishes in my order
```

Comment:
Metrics/LineLength: Line is too long. [106/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)